docs/docs/scanning.md

```
@@ -223,6 +223,114 @@ The default output format for a Kubescape scan is a "pretty-printed" table view.
+
+    ```sh
+    kubescape scan /path/to/manifests \
+    --hide \
+    --format json \
+    --output report.json
+    ```
+
+    !!! note "Note"
+       `--hide` replaces sensitive values with deterministic pseudonyms derived from an unsalted hash of the original value. Values drawn from a small or guessable set, such as common namespace names, can be recovered by hashing candidate values and matching the result, and identical values produce identical pseudonyms across reports.
+
+       Use `--hide` to reduce incidental exposure, not as a confidentiality guarantee. To share a report whose metadata is genuinely protected, use `--encrypt` and withhold the master key.
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)